src/server-middleware/healthcheck.js

```diff
@@ -1,4 +1,52 @@
-export default function healthcheck(req, res) {
-  res.setHeader('Content-Type', 'text/plain')
-  res.end('OK')
+import osu from 'node-os-utils'
+
+/**
+ * A Healthcheck which passes if the amount of avaliable memory
+ * is greater than our desired minimum avaliable memory percentage.
+ *
+ * Useful in the event of memory leaks, in particular.
+ *
+ * @type {import('@nuxt/types').ServerMiddleware}
+ */
+export default function Healthcheck(_, res) {
+  osu.mem
+    .info()
+    .then((info) => {
+      const avaliableMemory = info.freeMemPercentage
```

Comment:
Can we use `async/await` here or is it not allowed for some reason?

Reply:
I can check and switch it over, I'm not sure.

Reply:
done, works fine.